cms/static/js/views/pages/container_subviews.js

```diff
@@ -527,7 +527,7 @@ function($, _, gettext, BaseView, ViewUtils, XBlockViewUtils, MoveXBlockUtils, H
                 tagValueElement.className = 'tagging-label-value';
 
                 tagContentElement.appendChild(tagValueElement);
-                parentElement.appendChild(tagContentElement);
+                parentElement?.appendChild(tagContentElement);
```

Comment:
[question] I'm not sure that optional chaining which was added in ES2020 will build and work correctly in edx-platform.
```suggestion
                if (parentElement) {
                    parentElement.appendChild(tagContentElement);
                }
```

Reply:
I noticed that the edX platform already uses modern ES syntax quite effectively (for [example](https://github.com/openedx/edx-platform/blob/cc019fe99246950457fc8414248308764e66e0f2/cms/static/js/views/pages/container_subviews.js#L192)), and it works. I have added the change you suggested.